ui/menu.js

```diff
@@ -55,85 +56,84 @@ return $.widget( "ui.menu", {
 		this.mouseHandled = false;
 		this.element
 			.uniqueId()
-			.addClass( "ui-menu ui-widget ui-widget-content" )
-			.toggleClass( "ui-menu-icons", !!this.element.find( ".ui-icon" ).length )
 			.attr({
 				role: this.options.role,
 				tabIndex: 0
 			});
 
 		if ( this.options.disabled ) {
-			this.element
-				.addClass( "ui-state-disabled" )
-				.attr( "aria-disabled", "true" );
+			this._addClass( "ui-state-disabled" );
+			this.element.attr( "aria-disabled", "true" );
 		}
 
-		this._on({
-			// Prevent focus from sticking to links inside menu after clicking
-			// them (focus should always stay on UL during navigation).
-			"mousedown .ui-menu-item": function( event ) {
-				event.preventDefault();
-			},
-			"click .ui-menu-item": function( event ) {
-				var target = $( event.target );
-				if ( !this.mouseHandled && target.not( ".ui-state-disabled" ).length ) {
-					this.select( event );
-
-					// Only set the mouseHandled flag if the event will bubble, see #9469.
-					if ( !event.isPropagationStopped() ) {
-						this.mouseHandled = true;
-					}
+		this._addClass( "ui-menu", "ui-widget ui-widget-content" )
+			._toggleClass( "ui-menu-icons", null, !!this.element.find( ".ui-icon" ).length )
```

Comment:
Existing issue, should probably address it here anyway: This call duplicates parts of `refresh()`, which is called later in this method. Maybe there is a reason for the duplication, but I guess that its just an oversight. If so, this line should be dropped.